app/salaries/routes/salaries.py

```diff
@@ -13,6 +13,7 @@
     "/api/salaries",
     summary="Get salaries information.",
     response_model=SalaryOut,
+    tags=["Salaries"]
```

Comment:
this is a test comment, here i can tell you if you need to change an specific line of code.